scipy/optimize/tnc.py

```diff
@@ -218,6 +218,17 @@ def fmin_tnc(func, x0, fprime=None, args=(), approx_grad=0,
     constraint. The specific constraint removed is the one
     associated with the variable of largest index whose
     constraint is no longer active.
+    
+    Return codes are defined as follows ::
```

Comment:
Can you remove the space from ``follows ::`` and insert a blank line below it? Then it will build correctly for the html/pdf docs.